### PR TITLE
lvol/blob: get shallow copy status of ended ops

### PR DESCRIPTION
**Pull request by DamiaSan**
Shallow copy status RPC now returns info of ongoing and even of ended operations.

### CODE REVIEW

**Review by innobead — APPROVED** [review]
LGTM.

Waiting for @shuo-wu testing before merging this PR.

cc @derekbit

**Comment by innobead** [comment]
Just noticed some responses from other RPC methods also use `state` with human-readable string for its value. 
https://github.com/longhorn/spdk/blob/c22a9a76836ca43d6298a96340e70a83f089d5e0/doc/jsonrpc.md#L10129

File: module/bdev/lvol/vbdev_lvol_rpc.c [hunk 1]
```diff
@@ -1474,16 +1475,26 @@ rpc_bdev_lvol_shallow_copy_status(struct spdk_jsonrpc_request *request,
...
 	w = spdk_jsonrpc_begin_result(request);
 
 	spdk_json_write_object_begin(w);
-	spdk_json_write_named_bool(w, "in_progress", total_clusters > 0);
-	if (total_clusters > 0) {
-		spdk_json_write_named_string_fmt(w, "status", "%lu/%lu", cluster_index, total_clusters);
+
+	spdk_json_write_named_string_fmt(w, "progress", "%lu/%lu", copied_clusters, total_clusters);
+	if (result > 0) {
+		spdk_json_write_named_string(w, "state", "none");
```
Comment (shuo-wu) [inline module/bdev/lvol/vbdev_lvol_rpc.c:1488]: When will this `result` become a positive number?

**Review by shuo-wu — APPROVED** [review]
In general LGTM

File: module/bdev/lvol/vbdev_lvol_rpc.c (hunk 1 above)
Comment (innobead) [reply module/bdev/lvol/vbdev_lvol_rpc.c:1488]: After initialization, https://github.com/longhorn/spdk/blob/7e86ff9d390020e5efbb5d423f47bca26058d81f/lib/blob/blobstore.c#L307

Comment (DamiaSan) [reply module/bdev/lvol/vbdev_lvol_rpc.c:1488]: At the beginning it is initialized to 1, so I can check that non operations have been made
In blob_alloc:
```
blob->u.shallow_copy.bserrno = 1;
```

**Pull request merged**